01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+[*1..20].each do |n|
```

Comment:
わざわざ配列にしなくてもrangeをeachしちゃえばOKです。
```suggestion
(1..20).each do |n|
```